core/Metrics/Formatter.php

```diff
@@ -72,10 +72,11 @@ public function getPrettyTimeFromSeconds($numberOfSeconds, $displayTimeAsSentenc
 
         // Display 01:45:17 time format
         if ($displayTimeAsSentence === false) {
-            $hours   = floor($numberOfSeconds / 3600);
-            $minutes = floor(($reminder = ($numberOfSeconds - $hours * 3600)) / 60);
+            $days    = floor($numberOfSeconds / 86400);
+            $hours   = floor(($reminder = ($numberOfSeconds - $days * 86400)) / 3600);
+            $minutes = floor(($reminder = ($reminder - $hours * 3600)) / 60);
             $seconds = floor($reminder - $minutes * 60);
-            $time    = sprintf("%02s", $hours) . ':' . sprintf("%02s", $minutes) . ':' . sprintf("%02s", $seconds);
+            $time    = sprintf("%02s", $days) . ':' . sprintf("%02s", $hours) . ':' . sprintf("%02s", $minutes) . ':' . sprintf("%02s", $seconds);
             $centiSeconds = ($numberOfSeconds * 100) % 100;
             if ($centiSeconds) {
```

Comment:
Should we maybe display days only if there was actually a day? Also it formats a time here, I'm not sure if  eg `01:01:01:01` would be clear that the first one is actually meant as days? Not sure how one would format something like this. Maybe we'd need to generate full date in this case? eg something like `2015-01-01 01:01:01`?